src/js/fluxcontroller.js

```diff
@@ -927,7 +927,7 @@ define(function (require, exports, module) {
                 debouncedName = name + DEBOUNCED_ACTION_SUFFIX,
                 synchronizedAction = this._synchronize(namespace, module, name),
                 throttledAction = synchronization.throttle(synchronizedAction),
-                debouncedAction = synchronization.debounce(synchronizedAction);
+                debouncedAction = _.debounce(synchronizedAction);
```

Comment:
Re: @mcilroyc's concerns about zero-delay debouncing, which I completely agree with, I wonder if we shouldn't just take this opportunity to remove these auto-debounced actions as well.